code/design_pattern/src/facade/Facade

```diff
@@ -0,0 +1,6 @@
+The whole purpose of Facade is to "aggregate" multiple classes and functionality into a single place, thus acting like
```

Comment:
Please use a simplified version of our [documentation guide](https://github.com/OpenGenus/cosmos/blob/master/guides/documentation_guide.md).